code/datums/diseases/advance/symptoms/stimulant.dm

```diff
@@ -1,38 +0,0 @@
-/*
-//////////////////////////////////////
-
-Stimulant //gotta go fast
-
-	Noticable.
-	Lowers resistance significantly.
-	Decreases stage speed moderately..
-	Decreases transmittablity tremendously.
-	Moderate Level.
-
-Bonus
-	The body generates Ephedrine.
-
-//////////////////////////////////////
-*/
-
-/datum/symptom/stimulant
-
-	name = "Stimulant"
-	stealth = -1
-	resistance = -3
-	stage_speed = -2
-	transmittable = -4
-	level = 3

```

Comment:
I agree with you. But it would require a rewrite of how it functions fundamentally, given that all the symptom does is give you shots of ephedrine.